mkosi.build

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh -x
+
+# This file is part of systemd.
+#
+# Copyright 2016 Lennart Poettering
+#
+# systemd is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# systemd is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with systemd; If not, see <http://www.gnu.org/licenses/>.
+
+# This is a build script for OS image generation using mkosi (https://github.com/systemd/mkosi).
+# Simply invoke "mkosi" in the project directory to build an OS image.
+
+git clean -dfqx
```

Comment:
OK. This wasn't obvious before reading mkosi source code. This version is certainly good enough for now, even though it may copy too much stuff around. Maybe mkosi should use something that is strictly tied to git, e.g. `git worktree add $DESTDIR/something`. Dunno, just speculating here.